admin/configuration/login.md

```diff
@@ -0,0 +1,25 @@
+# User authentication & authorization
+
+(admin/configuration/authentication)=
+## Authentication
+
+Users can prove who they are by logging in via an *authentication provider*. Currently, the following providers are supported:
+
+1. *Google*. This includes public `@gmail.com` accounts, as well as [Google Workspace](https://workspace.google.com/) accounts set up for your workspace or university. If you use the GMail interface to access your work / university email, it can be used here.
+
+2. [*GitHub*](https://github.com/). Extremely popular community of people creating, publishing and collaborating on code. Accounts are free, and many people already have them especially since the target community for most hubs are people who also write some kind of code.
+
+3. [*ORCID*](https://orcid.org/). Everyone who has published a paper has one of these, and anyone else can easily sign up. Almost exclusively used by researchers.
+
+4. ???. We could probably support other authentication providers, depending on your specific needs and the provider's complexity. Please reach out to us if none of these 3 work.
+
+## Authorization
```

Comment:
I think we could have two sections: Authorizing Administrators, and Authorizing Users. Then we explain that administrators are generally authorized via configuration, while users are authorized via the admin panel.

Reply:
I've addressed this without splitting it into sections, how does it look?

Reply:
I think it's fine, but could we add in "regular users" and "admin users" in **bold** rather than regular text? I think it's important for documentation to be glance-able. Most people will not read the whole page, they'll have a specific question in mind and they'll quickly glance the page for cues that the answer is there. So I think section headers, or bold text, is a way to give people a hint where a particular topic is covered. Does that make sense?